### PR TITLE
Add unit test for InstanceContext

### LINKED ISSUE #15051: Add unit test for InstanceContext

### PR DESCRIPTION
**Pull request by csonezp**
Fixes #15051.
Add unit test for InstanceContext

### CODE REVIEW

File: shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/instance/InstanceContextTest.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+public final class InstanceContextTest {
+
+    private final ModeConfiguration modeConfiguration = new ModeConfiguration("Memory", null, false);
+
+    @Test
+    public void assertUpdateInstanceStatus() {
+        InstanceContext context = new InstanceContext(new ComputeNodeInstance(), new WorkerIdGeneratorFixture(Long.MIN_VALUE), modeConfiguration);
+        StateType actual = context.getState().getCurrentState();
+        assertThat(actual, is(StateType.OK));
+        //circuit break
```
Comment (menghaoranss) [inline shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/instance/InstanceContextTest.java:44]: please remove unnecessary comment.

File: shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/instance/InstanceContextTest.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        context.updateInstanceStatus(Lists.newArrayList(StateType.CIRCUIT_BREAK.name()));
+        actual = context.getState().getCurrentState();
+        assertThat(actual, is(StateType.CIRCUIT_BREAK));
+        //online
```
Comment (menghaoranss) [inline shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/instance/InstanceContextTest.java:48]: please remove unnecessary comment.

**Review by menghaoranss — CHANGES_REQUESTED** [review]

File: shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/instance/InstanceContextTest.java (hunk 1 above)
Comment (csonezp) [reply shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/instance/InstanceContextTest.java]: done

File: shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/instance/InstanceContextTest.java (hunk 2 above)
Comment (csonezp) [reply shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/instance/InstanceContextTest.java]: done

**Review by menghaoranss — APPROVED** [review]

**Pull request merged**